File: Python Word Examples/TextRangesApiDemo/updateTextRangeText.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TextRangesApi(configuration)

file_name = 'Template.docx'
paragraph_path = "Section/0/Body/0/Paragraph/0"
index = 0
text = 'E-iceblue test textRange'
extra = {'folder': "input",'dest_file_path': 'output/UpdateTextRangeText.docx'}
result =api.update_text_range_text(file_name, paragraph_path, index, text, **extra)

File: JavaScript Word Examples/TextRangeApiDemo/updateTextRangeText.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TextRangesApi(apiClient);

    var name = "Template.docx";
    var paragraphPath = "Section/0/Body/0/Paragraph/0";
    var index = 0;
    var text = "E-iceblue test textRange";
    var destFilePath = "output/UpdateTextRangeText.docx";
    var opts = {
        folder: 'input',
        storage: null,
        password: null
    };
    instance.updateTextRangeText(name, paragraphPath, index, text, destFilePath, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();